source/reference/command/setParameter.txt

```diff
@@ -18,20 +18,40 @@ setParameter
    Replace the ``<option>`` with one of the supported
    :dbcommand:`setParameter` options:
 
-   - :parameter:`journalCommitInterval`
+   **Indexing**
+
+   - :parameter:`failIndexKeyTooLong`
+   - :parameter:`notablescan`
+
+   **Logging**
+
+   - :parameter:`logComponentVerbosity`
    - :parameter:`logLevel`
    - :parameter:`logUserIds`
-   - :parameter:`notablescan`
    - :parameter:`quiet`
+   - :parameter:`traceExceptions`
+
+   **Replication**
+
    - :parameter:`replApplyBatchSize`
    - :parameter:`replIndexPrefetch`
-   - :parameter:`syncdelay`
-   - :parameter:`traceExceptions`
-   - :parameter:`textSearchEnabled`
-   - :parameter:`sslMode`
+
+   **Security**
+
+   - :parameter:`auditAuthorizationSuccess`
    - :parameter:`clusterAuthMode`
+   - :parameter:`enableLocalhostAuthBypass`
```

Comment:
I think enableLocalhostAuthBypass is only set when starting up since in order to be able to setParameter, you'll need to be able to get in first.